src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicRegexNode.cs

```diff
@@ -77,7 +78,7 @@ private static SymbolicRegexNode<TSet> Create(SymbolicRegexBuilder<TSet> builder
             var key = (kind, left, right, lower, upper, set, info);
             if (!builder._nodeCache.TryGetValue(key, out SymbolicRegexNode<TSet>? node))
             {
-                node = new SymbolicRegexNode<TSet>(kind, left, right, lower, upper, set, info);
+                node = new SymbolicRegexNode<TSet>(builder, kind, left, right, lower, upper, set, info);
 #if DEBUG
                 node._debugBuilder = builder;
 #endif
```

Comment:
I guess this could be moved into the ctor now?

Reply:
Yes, good catch :)